Modules/Wiki/classes/class.ilWikiPageGUI.php

```diff
@@ -141,7 +141,7 @@ public function executeCommand() : string
             case "ilcommonactiondispatchergui":
                 $gui = ilCommonActionDispatcherGUI::getInstanceFromAjaxCall();
                 $gui->enableCommentsSettings(false);
-                $gui->setRatingCallback($this, "preview");
+                $gui->setRatingCallback($this, "preview");// TODO PHP8-REVIEW This cannot work because the type defined in \ilCommonActionDispatcherGUI::setRatingCallback is `ilObjectGUI`
```

Comment:
This cannot work because the type defined in \ilCommonActionDispatcherGUI::setRatingCallback is `ilObjectGUI`